xmltojsonlegacy/src/main/java/org/assimbly/xmltojsonlegacy/processor/XmlToJsonProcessor.java

```diff
@@ -7,6 +7,7 @@
 import org.apache.log4j.Logger;
```

Comment:
it would be good to separate the processing (in this class) from the configuration (in the other mentioned classes)

Reply:
created 2 separated components: xmltojsonlegacy and jsontoxmllegacy